### DISCUSSION TITLE
Why does filament use a separate User model?

### DISCUSSION THREAD

**Discussion by tanthammar** [filamentphp/filament#67] [Help] [answered]
I'm interested to know why Filament doesn't use the normal User model?
Not complaining, but a bit confused by the choice with an additional User Class with a separate user and password_reset tables.

Just installed the package, created a Filament user and immediately got an error in my UserObserver because it doesn't receive a normal User model, but a FilamentUser.

Example error:
![image](https://user-images.githubusercontent.com/21239634/109825449-21af0e00-7c3a-11eb-81b8-304baeedfd56.png)

With very little knowledge of the package this makes me worry that I have to cover for different User models throughout the app.

**Comment by danharrin** [comment]
Hey @tanthammar. I've opened #72 with support for this feature. Would be great if you could review, thanks!

**Comment by danharrin** [comment, marked as answer]
Supported from v1.3.0.
